Put all annotations inline instead of above (#188)
runtimeverification/vlsm: CODING_CONVENTIONS.md

}.
```

### Locality annotations

Locality annotations should be put on the same line as the command they annotate. This makes it easier to process them with line-based tools, like `grep` or `sed`.

Good:
```coq
#[export] Instance compare_eq_dec {A} `{CompareStrictOrder A} : EqDecision A.
```

> Bad:

Reviewer: I don't like the terminology "bad", and there are many places where annotations will make the line extremely long.
```suggestion
Not recommended:
```

```coq
#[export]
Instance compare_eq_dec {A} `{CompareStrictOrder A} : EqDecision A.
```

> 

Reviewer: Can you please come up with some rule of thumb for how to handle long lines with annotations and `Instance`, for example? It's natural to place the linebreak after the annotation.

Reviewer 2: That seems to be forced if we want line-based tools to be able to connect the annotation with the definition name.

}.
```

### Locality annotations

Locality annotations should be put on the same line as the command they annotate. This makes it easier to process them with line-based tools, like `grep` or `sed`.

Recommended:
```coq
#[export] Instance compare_eq_dec {A} `{CompareStrictOrder A} : EqDecision A.
```

Not recommended:
```coq
#[export]
Instance compare_eq_dec {A} `{CompareStrictOrder A} : EqDecision A.
```

> In case you need to split the line because it's too long, keep the annotation on the same line as the command and the name. When splitting the above example:

Reviewer: ```suggestion
In case you need to split a long line, keep the annotation on the same line as the command and the name. When splitting the above example:
```

Author: Done.
